modules/user_accounts/jsx/reject_users.js

```diff
@@ -0,0 +1,18 @@
+$(document).ready(
```

Comment:
not sure who is taking over this PR, but this should not be in the `jsx` directory since it is not jsx
it also does not need to be included in webpack.config.js for the same reason
(hi leo if you are reading this hope you are doing well)